src/replication.c

```diff
@@ -2712,22 +2720,26 @@ int sendCurrentOffsetToReplica(client *replica) {
     return C_OK;
 }
 
+char *replicationSendAuth(connection *conn) {
```

Comment:
This returns an sds, but I see sendCommandArgv() also is lying about it's type. We should probably fix it.